clients/js/networks.ts

```diff
@@ -307,6 +329,8 @@ const DEVNET = {
   near: {
     rpc: undefined,
     key: undefined,
+    networkId: "sandbox",
+    baseAccount: "test.near",
```

Comment:
couldn't this be `wormhole.devnet` or some such?

Reply:
This is what Josh chose.